extensions/stackdriver/config/v1alpha1/stackdriver_plugin_config.proto

```diff
@@ -120,4 +120,11 @@ message PluginConfig {
   // logs. Conflicts are resolved by the tag name by overriding previously
   // supplied values. Does not apply to audit logs.
   CustomConfig custom_log_config = 14;
+
+  // Optional. Controls the metric expiry duration. If a metric time series is
+  // not updated for the given duration, it will be purged from time series
+  // cache as well as metric reporting. If this is not set or set to 0, time
+  // series will never be expired. This option is useful to avoid unbounded
```

Comment:
Add comment?: a high value here means higher memory consumption, but potentially also a higher tail latency ( if it is true), when purging metrics, does a big purge cost more thAn small purge for cpu? If we are iterating over the whole set, it may not matter.

Reply:
Especially true for cloud run use case.

Reply:
Opencensus uses its own thread pool, so I think it won't affect tail latency?